vespa-cloud/vespa-documentation-search/src/test/java/ai/vespa/cloud/docsearch/VespaDocSystemTest.java

```diff
@@ -121,6 +165,14 @@ public String getContinuationElement(String visitResult) throws Exception {
                 .POST(ofString(doc.toString())));
     }
 
+    public HttpResponse<String> updateTestDoc(JsonNode update) {
+        return testEndpoint.send(testEndpoint
+                .request("/document/v1/open/doc/docid/" +
+                                "open"+ update.get("fields").get("path").get("assign").textValue(),
```

Comment:
Hmm... Doesn't this mean the old document lives on, as the update also indicates a new doc id?

Reply:
this updates the current doc - must get the ID from the update operation, but the id stays the same

Reply:
Doesn't the update operation have the id as a separate thing? You just don't want to parse the docId, I guess?